resources/sdk/pureclouddotnet/templates/ApiClient.mustache

```diff
@@ -248,6 +249,9 @@ namespace {{packageName}}.Client
                 }
             }
 
+            if ((int)response.StatusCode < 200 || (int)response.StatusCode >= 300)
```

Comment:
Why do you have to do the cast here.  Doesnt .NET do autoboxing of values.  If not I would be surprised if their Integer object does not have a toInt() method. on it.